src/content/overview/developer-portal/_index.md

```diff
@@ -6,7 +6,27 @@ menu:
   principal:
     parent: overview
     identifier: overview-developer-portal
-last_review_date: 2024-03-18
+last_review_date: 2024-06-11
 owner:
   - https://github.com/orgs/giantswarm/teams/sig-product
 ---
+
+The Giant Swarm Developer Portal is the central self-service portal for your platform engineers and developers. It provides access to all the tools and resources needed to manage and develop applications on your Giant Swarm platform.
+
+Giant Swarm uses [Backstage](https://www.cncf.io/projects/backstage/) as the foundation for the developer portal. Backstage is an open-source platform for building developer portals. It provides a unified view of all your software development tools and services, navigate across the documentation even creating custom plugins to integrate with your existing tools.
+
+## Managed capabilities
+
+The Developer Portal provides the following features:
+
+- **Service catalog**: a catalog of all the services available on your platform, including documentation, metrics, and other information.
+- **Application management**: the ability to create golden paths or template to boost the application development process and the same time you can observe the status of your applications.
```

Comment:
```suggestion
- **Application management**: the ability to create golden paths or templates to boost the application development process and the same time enable you to observe the status of your applications.
```

Reply:
I'd like to also add sth that is not directly application related, but we see often times a use case within end users:
Basically, it's about the platform teams offering (and in turn developers consuming) self-service utilization of certain environments and common services and (cloud) resources, typically:
- Request a new environment/namespace for your team/project
- Request a managed DB for your project
...

Maybe that needs to be a separate point. WDYT?